src/motor_instructions.py

```diff
@@ -63,7 +50,11 @@ def send_instruction(self, instr):
 def init_subscribers():
     rospy.Subscriber("x_center", Float32, pan_callback)
     rospy.Subscriber("y_center", Float32, tilt_callback)
+    rospy.Subscriber("state", Int32, state_callback)
 
+def state_callback(state):
+    rospy.loginfo(state)
+    rospy.loginfo("Is working")
```

Comment:
The state doesn't seem to be affecting any of the code in this node. It looks like you fixed this in a more recent commit though.